### ISSUE TITLE
adding this to wlan1?

### ISSUE THREAD

**Issue by JohnModica** [cryptolok/GhostInTheNet#5]
I tried to edit this to start on wlan1. no luck.. would you mind writing a copy or editing a copy to use wlan1 function? it would say wlan1 not active

**Comment by cryptolok** [comment]
you don't need to edit the script, you just specify the interface to it as argument:
`GhostInTheNet.sh on wlan1`
anyway, can you please paste the whole output along with your net confs? like `ip a` or `sudo ifconfig -a`
I imagin that you're trying to do it with an external WiFi card? like Alfa, in that case, you should be already authenticated to a network, since I use NetworkManager to connect automatically, but that's not really a problem, but the MAC address may not change though

**Comment by JohnModica** [comment]
Why is this closed?
I have done this exact commands since before you even comment. Still doesn't work. I'm not a dope, since I have written numerous scrips before.
I have used macchanger and without macchanger.
Your script just doesn't work.
Doesn't work for eth0 either.

**Comment by cryptolok** [comment]
it was closed since, you didn't provide any information I was asking that could help to solve the issue, if "you have done this exact commands since before I even comment", too bad you're not willing to share the result, like your OS, networking and stuff
I'm not questioning your intellectual abilities, but if you clame your self a shell guru, you would solve the issue your self in the first place :)
and finally, my script works, especially for ethernet interfaces
please note, that I have changed my script consecutively

TLDR; collaborate or deal with it
